accounts/templates/accounts/participant_detail.html

```diff
@@ -1,42 +1,289 @@
-              {{ study.name }},
-            {% endfor %}
-          </td>
-        </tr>
-        <tr>
-            <th>Responses</th>
-            <td>
-                {% for resp in user.response_set.all %}
-                    <a href="{% url 'exp:response-detail' resp.id %}">{{ resp.study.name }}</a><br/>
+  <div class="container mb-lg">
+      <div class="row">
+          <div class="col-xs-12">
+              <a href="{% url 'exp:participant-list' %}"> All Participants </a> > {{ user.get_short_name }} </a>
+          </div>
+      </div>
+
+      <div class="row">
+          <div class="col-xs-12">
+              <h1> {{ user.get_full_name }} </h1>
+              <span class="pr-md"><label class='pr-xs'> Email: </label> {{ user.username }} </span>
+              <span><label class='pr-xs'> Last Active: </label> {{ user.last_login|date:"n/d/Y"|default:"N/A" }} </span>
+          </div>
+      </div>
+
+    <div class='row'>
+        <div class='col-sm-6'>
+            <h3>Children</h3>
+            {% if children %}
+                {% for child in children %}
+                <div class="pb-md">
+                    <h4 class="child-name"><em>{{child.name}}</em></h4>
+                    <div class="pl-md">
+                        <div class="row">
+                            <div class="col-xs-5">
+                                <b> Birthday: </b>
+                            </div>
+                            <div class="col-xs-7">
+                                {{child.birthday|date:"n/d/Y"|default:"N/A"}}
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-xs-5">
+                                <b> Gender: </b>
+                            </div>
+                            <div class="col-xs-7">
+                                 {{child.gender}}
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-xs-5">
+                                <b> Gestational Age at Birth: </b>
+                            </div>
+                            <div class="col-xs-7">
+                                 {{child.age_at_birth}}
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-xs-5">
+                                <b> Age: </b>
+                            </div>
+                            <div class="col-xs-7">
+                                 {{child.age}}
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-xs-5">
+                                <b> Extra: </b>
+                            </div>
+                            <div class="col-xs-7">
+                                 {{child.extra}}
```

Comment:
This "extra" field is incorrectly named.  It's actually the additional_information field on the child which is a string, and was renamed to "extra" in get_context_data.  We don't need the <pre> here